khtools/bloom_filter.py

```diff
@@ -0,0 +1,253 @@
+import math
+import os
+
+import click
+import khmer
+import screed
+from sourmash._minhash import hash_murmur
+from tqdm import tqdm
+
+from khtools.compare_kmer_content import kmerize
+from khtools.sequence_encodings import encode_peptide, VALID_PEPTIDE_MOLECULES
+
+# khmer Nodegraph features
+DEFAULT_N_TABLES = 4
+DEFAULT_MAX_TABLESIZE = int(1e8)
+
+# Default k-mer sizes for different alphabets
+DEFAULT_PROTEIN_KSIZE = 7
+DEFAULT_DAYHOFF_KSIZE = 11
+DEFAULT_HP_KSIZE = 21
```

Comment:
instead of hardcoding these values maybe add them to a config file instead, that way it will be easier for a user to track down defaults for debugging.